### ISSUE TITLE
cluster namespaces conflicts when using same deis login 

### ISSUE THREAD

**Issue by jayaprakash369** [deis/deis#5127]
I had created two EKS cluster and in one cluster installed deis workflow and deployed deis apps (which is used for staging).

after that created another EKS cluster and login with same deis user id and created another deis admin user.

From that created admin user I connected second EKS cluster and tried to create same app name as in staging(for dev purpose) and facing,
```Creating Application... Error: Application with this id already exists.```

what is conflicting here. how can I resolve this?

can I create same apps for different users with same deis login?

deis version: v2.22.3

helm version: 
```version.BuildInfo{Version:"v3.5.1", GitCommit:"32c22239423b3b4ba6706d450bd044baffdcf9e6", GitTreeState:"clean", GoVersion:"go1.15.7"}```

kubectl version:
``` Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.3", GitCommit:"c92036820499fedefec0f847e2054d824aea6cd1", GitTreeState:"clean", BuildDate:"2021-10-27T18:41:28Z", GoVersion:"go1.16.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19+", GitVersion:"v1.19.13-eks-8df270", GitCommit:"8df2700a72a2598fa3a67c05126fa158fd839620", GitTreeState:"clean", BuildDate:"2021-07-31T01:36:57Z", GoVersion:"go1.15.14", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.22) and server (1.19) exceeds the supported minor version skew of +/-1 
```

**Comment by bacongobbler** [comment]
This project has been inactive for over half a decade.

**Comment by jayaprakash369** [comment]
> This project has been inactive for over half a decade.

is there no any other update in deis project since half a decade?
is there no support for deis now?
we stucked in one project which was deployed before 3 yrs ago using deis workflow. now we need to migrate the same deis apps from kops cluster to EKS cluster. we are facing so many issues.

**Comment by bacongobbler** [comment]
Yes. The last release for Deis Workflow was back in September 2017 before the project entered maintenance mode. Maintenance mode ended March 2018. We were [acquired by Microsoft](https://blogs.microsoft.com/blog/2017/04/10/microsoft-acquire-deis-help-companies-innovate-containers/) back in April 2017.

> deis version: v2.22.3

[We never a released v2.22.3](https://github.com/deis/workflow/releases). The last release was 2.18.0.

Are you referring to the hephy fork? https://github.com/teamhephy/workflow
